scss/_navbar.scss

```diff
@@ -419,3 +419,29 @@
     }
   }
 }
+
+.supra {
+  padding-bottom: $navbar-supra-padding-bottom;
+  border-bottom: 1px solid $gray-700;
```

Comment:
Are you sure about this color value? It was `#333` in v4 and is set to `#666` here.
If `#666` is in fact the good one, we need to think to change it in v4.

Reply:
I'm quite sure about it
![image](https://user-images.githubusercontent.com/5867791/133749271-02005a5f-9715-4a42-8a51-6d81a73b5eb0.png)

And by the way I noticed I didn't put it at the bottom on every single global header, which should be the case...

Reply:
Just to track this, an issue has been opened #804